app/controllers/publishers/vacancies_controller.rb

```diff
@@ -8,6 +8,9 @@ def show
 
   def create
     vacancy = Vacancy.create
```

Comment:
You can do that in one call
`vacancy = Vacancy.create(publisher: current_publisher, publisher_organisation: current_organisation)`